scripts/tools.sh

```diff
@@ -43,7 +43,7 @@ then
   # Disabled due to issues with windows curl command not working
   # Related to https://github.com/keboola/keboola-as-code/pull/1818/commits/df23a4e2855557a01460593f7d6ba59b5dc8825c
   # curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$GOBIN" v1.58.0
-  go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.57.2
+  go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
```

Comment:
Can we put there `v1.61.0`? When they do update our CI will return error. We should do the update once a month, otherwise we will do once a month PR for update linter due to CI fails.